index.html

```diff
@@ -4,13 +4,67 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Instagram Clone</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="styles/style.css"/>
   </head>
   <body>
-    Instagram Capture and Share the World's Moments Instagram is a fast, beautiful and fun way to share your life with
-    friends and family. Take a picture or video, choose a filter to transform its look and feel, then post to Instagram
-    &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-    Oh yeah, did we mention it's free? Download on the App Store Get it on Google Play About Us Support Blog Press API
-    Jobs Privacy Terms &copy; 2014 Instagram
+
+    <section class="s1">
+
+      <article class="a1">
+        <img class="phones" src= "images/phones.png" alt = "phones"/>
+      </article>
+
+      <article class="a2">
+          <div class="diva1">
+            <img src="images/brand.png" alt="Instagram">
+            <div class="diva11">
+              <img src="images/home.png" alt="home">
+              <button class="buttondiva1">log in </button>
```

Comment:
Este login aunque parece un botón en realidad es un enlace. Debería ser algo en este estilo:

`<a class="login-button"><img src="images/home.png" alt="home">log in</a>`